_layouts/default.html

```diff
@@ -1,12 +1,10 @@
----
```

Comment:
🤔 i am thinking now&hellip;

Reply:
![image](https://github.com/OpenINF/openinf.github.io/assets/17770407/bdb3d2db-a4b9-46bf-8b86-dfb0f9e56425)

![image](https://github.com/OpenINF/openinf.github.io/assets/17770407/71a70f1f-4c63-4ee7-9130-a391d1fc65f7)

🔗 https://jekyllrb.com/docs/front-matter/

Reply:
so, set editable to "false" by default and we can set it to "true" by default for:

[Tutorials](https://cdpn.io/DerekNonGeneric/fullembedgrid/OJmxmKe/5f1aba04a72431dc166decc5d32fc773?animations=run&type=embed#section_01)
    [How-to guides](https://cdpn.io/DerekNonGeneric/fullembedgrid/OJmxmKe/5f1aba04a72431dc166decc5d32fc773?animations=run&type=embed#section_02)
    [Reference guides](https://cdpn.io/DerekNonGeneric/fullembedgrid/OJmxmKe/5f1aba04a72431dc166decc5d32fc773?animations=run&type=embed#section_03)
    [Explanations](https://cdpn.io/DerekNonGeneric/fullembedgrid/OJmxmKe/5f1aba04a72431dc166decc5d32fc773?animations=run&type=embed#section_04)